blueocean-core-js/package.json

```diff
@@ -23,13 +24,17 @@
     "url": "https://github.com/jenkinsci/blueocean-plugin.git"
   },
   "dependencies": {
-    "isomorphic-fetch": "2.2.1"
+    "isomorphic-fetch": "2.2.1",
+    "jsonwebtoken": "^7.1.9",
+    "pem-jwk": "^1.5.1"
```

Comment:
Can we use exact version numbers for dependencies? Later on we should probably add shrinkwrap.json for 100% reproducibility.